packages/web/src/scss/settings/_featureFlags.scss

```diff
@@ -0,0 +1 @@
+$enable-dropdown-enhanced-shadow: false !default;
```

Comment:
I'm wondering about putting component name at start:

```suggestion
$dropdown-enable-enhanced-shadow: false !default;
```

What do you think? 🤔